### PR TITLE
AC: Add detection labels shift postprocessing for Efficientdet

### PR DESCRIPTION
**Pull request by jkamelin**
converted Efficientdet model returns class labels shifted by 1 (starting from 0, where 0 is person label)

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/postprocessor/shift.py [hunk 1]
```diff
@@ -55,3 +55,34 @@ def process_image(self, annotation, prediction):
...
+        self.offset = self.get_value_from_config('offset')
+
+    def process_image(self, annotation, prediction):
+        for prediction_ in prediction:
+            prediction_.labels += self.offset
+
+        for annotation_ in annotation:
+            annotation_.labels += self.offset
+
+        return annotation, prediction
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/postprocessor/shift.py:88]: @jkamelin could you please describe why these changes need?
Is it impossible to align labels during annotation conversion?

**Comment by eaidova** [comment]
@jkamelin please add your postrocessor to readme

File: tools/accuracy_checker/accuracy_checker/postprocessor/shift.py (hunk 1 above)
Comment (jkamelin) [reply tools/accuracy_checker/accuracy_checker/postprocessor/shift.py:88]: These changes were made for uniformity, for this specific model, shift is only needed for predictions, in this case code for annotations can be removed. Do you suggest to perform align not in postprocessor, but during annotation conversion?

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/postprocessor/shift.py:85]: no, i think for now it is ok. If we change logic in converter, it will be not backward compatible

File: tools/accuracy_checker/accuracy_checker/postprocessor/shift.py [hunk 2]
```diff
@@ -55,3 +55,31 @@ def process_image(self, annotation, prediction):
...
+    __provider__ = 'shift_labels'
+
+    prediction_types = (DetectionPrediction, )
+    annotation_types = (DetectionAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'offset': NumberField(value_type=int, optional=True, default=0, description="Value for shift.")
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/postprocessor/shift.py:74]: I think we do not need default value here. with 0 it has not effect.

**Review by eaidova — APPROVED** [review]
